### PR TITLE
Bug 1709454: fixing missing RBAC for metadata-reader cluster role

### PR DESCRIPTION
**Pull request by ewolinetz**
making sure fluentd tests run and fixing so that fluentd is correctly installed

ref: https://bugzilla.redhat.com/show_bug.cgi?id=1709454

### CODE REVIEW

**Comment by ewolinetz** [comment]
@jcantrill we were missing permissions in our csv... not sure impact on this for getting it out for 4.1.0

File: manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -176,6 +176,7 @@ spec:
           - monitoring.coreos.com
           resources:
           - servicemonitors
+          - prometheusrules
```
Comment (nhosoi) [inline manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml]: Fluentd PrometheusRules were merged to 4.2 (master-post-release branch).  Is it ok to put it in the master branch?

Comment (ewolinetz) [reply manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml]: good catch, i just assumed it needed to go in without seeing if it failed without it. i'll verify this works for master even without it

**Comment by jcantrill** [comment]
@ewolinetz per our conversation, please open a bz for this to go into a 4.1.z for a 0 day.  I think you will have to open this against the 4.1 branch when it opens cc @bparees

**Pull request merged**